libs/algorithms/include/hpx/parallel/util/transform_loop.hpp

```diff
@@ -53,7 +53,7 @@ namespace hpx { namespace parallel { namespace util {
         {
             template <typename InIter1, typename InIter2, typename OutIter,
                 typename F>
-            HPX_HOST_DEVICE HPX_FORCEINLINE static hpx::util::tuple<InIter1,
+            HPX_HOST_DEVICE HPX_FORCEINLINE static util::in_in_out_result<InIter1,
                 InIter2, OutIter>
             call(InIter1 first1, InIter1 last1, InIter2 first2, OutIter dest,
```

Comment:
You may also want to adapt the begin/end iter template arguments for `transform_loop` too

Reply:
Got it! Thanks!